### ISSUE TITLE
Guidance installing the Urban Performance model in a-Shell

### ISSUE THREAD

**Issue by 2023me** [holzschu/a-shell#581]
Hello.

While I’ve used UNIX before, I’m completely new to it on the iPad/GitHub/etc. I’d like to install the Urban Performance model using this code:

https://github.com/UPTechMX/UPT-UrbanPerformance

I _think_ it’s possible. a-Shell seems to support all of the pieces. I just need some guidance getting them into place. My status so far…

1. Python is already installed, so that’s not an issue.
2. I’ve been able to install Django & Celery (I think).

I’m getting stuck at PostgreSQL. I found this in another GitHub page:

https://github.com/wasmerio/wasmer-postgres

but I’m uncertain as to what I need to do to use it in a-Shell. Also, the Urban Performance installation instructions say I need to:

sudo apt install postgresql-10 postgresql-10-postgis-3

but near as I know, a-Shell doesn’t have sudo. How do I handle this?

Thanks for your assistance, and consideration!

**Comment by holzschu** [comment]
I don't think it is possible to install django, celery and PostrgreSQL in a-Shell for the time being.

For such a complex task, I think your only choice is iSH. It will be slower, but you can install almost anything.

**Comment by 2023me** [comment]
That’s too bad. From what I can tell, it looks like celery and django are installed.

![IMG_1276](https://user-images.githubusercontent.com/127614546/224782003-7f3d3cf4-3178-4073-81ee-8af6c808eb5d.jpeg)

Alright.I will check out iSH.

Thanks for your assistance and consideration.
